src/app/features/notification/notification.component.html

```diff
@@ -92,8 +98,8 @@ <h2 class="notification-title" i18n>Notifications</h2>
     (click)="closeOnlySubmenu(settingsMenuTrigger, $event)"
   >
     <button mat-menu-item (click)="markAllRead()">
-      <fa-icon icon="check"></fa-icon>
-      <span class="menu-option" i18n>Mark all as read</span>
+      <fa-icon icon="check" class="standard-icon-with-text"></fa-icon>
```

Comment:
let's additionally change the overflow (a class "flex-row" around these might help). At the moment, if a translation is even longer than the German one, we still have the line break

Reply:
For longer translation, I am thinking like this (or also can show full text onhover) -
<img width="397" alt="image" src="https://github.com/user-attachments/assets/edbec789-c547-45c3-a2b5-0e413418503d" />

What's your thought?

Reply:
This looks good in my opinion. Let's go with this for now - should only be a safeguard for exceptional translations anyway